site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -4,145 +4,205 @@ name: Interest Invokers (Explainer)
-for more).
-
-Elements with `interesttarget` set will dispatch an `InterestEvent` on the
-_Interestee_ (the element referenced by `interesttarget`) when the element
-_Shows Interest_ or _Loses Interest_. When the element _Shows Interest_ the
-event type will be `'interest'`. If the element has _Shown Interest_, and
-interest moves away from both the _Interest Element_ and the _Interestee_, then
-the element _Loses Interest_ and an `InterestEvent` with the type of
-`'loseinterest'` will be dispatched on the _Interestee_. The event also contains
-a `invoker` property that will reference the _Interest_ element.
-`InterestEvent` objects are always non-bubbling, composed, cancellable events.
+The `interesttarget` value should be an IDREF pointing to an element within a single document or shadow root. The `interestTargetElement` IDL attribute also exists on the element to imperatively assign a node to be the invoker target, allowing for cross-root invokers (in some cases, see [attr-asociated element steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element) for more). We define this target element to be the "Interestee".
+
+Elements with an `interesttarget` attribute will dispatch an `InterestEvent` on the *Interestee* when the element
+*Shows Interest* or *Loses Interest*. When the user *Shows Interest* in the element, the event type will be `"interest"`. If the user has already shown interest in the element, and interest is subsequently lost, an `InterestEvent` with the type of `"loseinterest"` will be dispatched on the *Interestee*. The event also contains a `source` property that will reference the element with `interesttarget`. `InterestEvent` objects are always non-bubbling, composed, cancellable events.
 
 ```webidl
 [Exposed=Window]
 interface InterestEvent : Event {
-  constructor(DOMString type, InterestEventInit interestEventInit);
-  readonly attribute Element invoker;
-  readonly attribute DOMString type = "interest";
-  readonly attribute DOMString action;
+  constructor(DOMString type, optional InterestEventInit interestEventInit = {});
+  readonly attribute Element source;
 };
 dictionary InterestEventInit : EventInit {
-  DOMString action = "";
-  Element invoker;
+  Element? source = null;
 };
 ```
 
-Both `interesttarget` and `commandfor` can exist on the same element at the
-same time, and both should be respected.
+Both `interesttarget` and `commandfor` can exist on the same element at the same time, and both should be respected/functional, since they are activated using strictly separate actions by the user.
 
 If a `<button>` is a form participant, or has `type=submit`, then `commandfor`
 _must_ be ignored. `interesttarget` is still valid in these scenarios.
 
 If an `<input>` is a form participant, or has a `type` other than `reset`, `submit`, or
 `button`, then `interesttarget` _must_ be ignored.
```

Comment:
@keithamus do you know why this text is here? I don't actually see why `interesttarget` should be ignored anywhere. It's not possible to submit a form with this API.

Reply:
It's ignored on all non button inputs, is what this text reads to me. Though depending on #1138 we can probably just remove this line entirely.

Reply:
I've removed it, and closed #1138.

Reply:
This is probably copypasta from invokers.